ci-automation/vendor-testing/qemu.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+# Copyright (c) 2021 The Flatcar Maintainers.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+set -euo pipefail
+
+# Test execution script for the qemu vendor image.
+# This script is supposed to run in the SDK container.
```

Comment:
Do we need to couple this to the SDK container? Look at what we do for aarch64 - kola depends on a working qemu and dnsmasq, nothing more.

Reply:
This is for portability / reproducibility: any given version / release has its test environment pinned to a version that is derived from the git commit in `scripts` which corresponds to that version. I do understand aarch64 will need separate handling since there is no SDK for ARM currently.

Reply:
Instead of treating arm64 differently, it would be best to find an approach that works the same way for both. See also the comment from Kai:
> While the QEMU tests run inside the SDK container this was not the case for the other tests and I think it's good to have it that way because the kola binary that should be used is the one supplied by Jenkins and the SDK container would just add overhead to the test job.

Reply:
@jepio Your idea would be to use the `ghcr.io/kinvolk/kola-test-runner` image for all cases? That makes sense and can provide a similar reproducibility that @t-lo aims for here but with the two benefits that we support arm64 from day one and that we don't try to use kola from the SDK which will be outdated but expect it to exist in a folder (supplied by Jenkins or the user).

Reply:
I like this idea! Makes a lot of sense.

Reply:
We could wire up kola test runner container in `version.txt` (or another versionfile) the same way we already wire up the SDK version. The test scripts could be amended to use the test runner instead of the SDK container.